doc/content/GettingStarted/installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Installation"
```

Comment:
For consistency with the other files under "GettingStarted" shouldn't we be calling this "Install"?